Makefile.frankfurt

```diff
@@ -138,21 +144,30 @@ environ:
 # FIXME: nosetests is still using development.ini
 define build_templates
 	export DOCKER_IMG_LOCAL_TAG=${DOCKER_IMG_LOCAL_TAG} && \
-	export CURRENT_DIR=${CURRENT_DIR} && \
+	export CURRENT_DIRECTORY=${CURRENT_DIRECTORY} && \
 	envsubst < base.ini.in > base.ini && \
 	envsubst < dev.ini.in > dev.ini && \
 	envsubst < base.ini.in > production.ini && \
 	envsubst < dev.ini.in > development.ini && \
 	envsubst < local.ini.in > local.ini && \
 	envsubst < apache/wsgi-py3.conf.in > apache/wsgi.conf && \
-	envsubst <  apache/application.wsgi.in > apache/application.wsgi && \
+	envsubst < apache/application.wsgi.in > apache/application.wsgi && \
 	envsubst < docker-compose.yml.in > docker-compose.yml && \
 	envsubst < 25-mf-chsdi3.conf.in > 25-mf-chsdi3.conf
 endef
 
+local.ini: local.ini.in base.ini guard-OPENTRANS_API_KEY guard-PGUSER guard-PGPASSWORD guard-GIT_HASH_SHORT guard-APACHE_PORT
+	export CURRENT_DIRECTORY=${CURRENT_DIRECTORY} && \
+	export $(cat .env.default) && \
```

Comment:
is it correct to hard code this here to cat `.env.default` or would it make sense to change it to cat `ENV_FILE` here? If `ENV_FILE` is correct, probably it makes sense to `guard-` that as well?

Reply:
Commit [4ca402e](https://github.com/geoadmin/mf-chsdi3/pull/3896/commits/4ca402e8343834d2f0a3f1ed4068eaeace4ab222) made it dynamic.